ideascube/serveradmin/catalog.py

```diff
@@ -346,6 +361,15 @@ class StaticSite(SimpleZipPackage):
     typename = 'static-site'
     handler = Nginx
 
+    @property
+    def template_id(self):
+         if self.id == 'w2eu':
+             return self.id
+         template_id, extension = self.id.rsplit('.', 1)
+         if extension == 'map':
+             template_id = 'maps'
+         return template_id
```

Comment:
We could simplify a bit yes (by rename package to w2eu.<lang>)
But for maps, we use a single template for several packages and it seems wrong to me to rename all map packages to maps.<extension>

We agreed that template_id should be determined from the package kind. We are not doing it right now cause we still have few things to define (icons, ...).
Maybe we could delay the special cases fix to latter when we will fix the icon/theme/.. stuffs.